src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,102 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.dto.MatchResult;
+import lotto.enums.LottoRank;
+import lotto.util.InputHandler;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.List;
+import java.util.Map;
+
+public class LottoController {
+    private final LottoNumberGenerator lottoNumberGenerator;
+    private final ProfitCalculator profitCalculator;
+
+    public LottoController(LottoNumberGenerator lottoNumberGenerator, ProfitCalculator profitCalculator) {
+        this.lottoNumberGenerator = lottoNumberGenerator;
+        this.profitCalculator = profitCalculator;
+    }
+
+    public void run() {
+        PurchaseAmount purchaseAmount = readPurchaseAmount();
+        printPurchaseCount(purchaseAmount);
+        LottoTicket lottoTicket = LottoTicket.of(lottoNumberGenerator, purchaseAmount);
+        List<Lotto> ticket = lottoTicket.getTicket();
+
+        displayPurchasedLottoNumbers(ticket);
```

Comment:
1. ticket은 이 부분에서만 사용되는 것 같아요! 굳이 변수에 담지 않아도 되지 않을까요?
2. 각각의 변수를 사용하는 부분과 관련해서 공백을 나누거나 문제에서 주어진 출력 별로 공백을 나누어보면 어떨까요?

```suggestion
        PurchaseAmount purchaseAmount = readPurchaseAmount();
  
        printPurchaseCount(purchaseAmount);
        LottoTicket lottoTicket = LottoTicket.of(lottoNumberGenerator, purchaseAmount);
        displayPurchasedLottoNumbers(lottoTicket.getTicket());
```

Reply:
메서드명으로 ticket을 반환한다는 것을 알 수 있어서 말씀하신대로 변수를 따로 지정하지 않아도 될 것 같네요!
공백으로 관련 로직을 나누는 것에도 신경을 써야겠습니다 감사합니다!